packages/hyperflux/package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.5.3",
   "main": "index.ts",
   "description": "State Management for XREngine",
-  "private": true,
```

Comment:
unrelated, but I had to publish this package to fix install process on some machines